api/v1alpha1/clusterextension_types.go

```diff
@@ -485,15 +484,17 @@ type ClusterExtensionStatus struct {
 	//   - Reason: a string representation of the reason for the current state of the condition. Typically useful for building automation around particular Type+Reason combinations.
 	//   - Message: a human readable message that further elaborates on the state of the condition
 	//
-	// The current set of condition types are:
-	//   - "Installed", represents whether or not the package referenced in the spec.packageName field has been installed
+	// The global set of condition types are:
+	//   - "Installed", represents whether or not the a bundle has been installed for this ClusterExtension
 	//   - "Resolved", represents whether or not a bundle was found that satisfies the selection criteria outlined in the spec
-	//   - "Deprecated", represents an aggregation of the PackageDeprecated, ChannelDeprecated, and BundleDeprecated condition types.
-	//   - "PackageDeprecated", represents whether or not the package specified in the spec.packageName field has been deprecated
-	//   - "ChannelDeprecated", represents whether or not the channel specified in spec.channel has been deprecated
-	//   - "BundleDeprecated", represents whether or not the bundle installed is deprecated
 	//   - "Unpacked", represents whether or not the bundle contents have been successfully unpacked
 	//
+	// When the ClusterExtension is sourced from a catalog, the following conditions are also possible:
```

Comment:
I grouped the `Deprecated*` conditions into their own section for two reasons:
1. I think it is helpful to see the other "main" conditions without the "noise" of these being intermingled.
2. Deprecation metadata comes from catalogs, and while we currently only support sourcing from catalogs, I thought it might be good to call out the fact that deprecations will only ever show up if the ClusterExtension is sourced from a catalog.